server/app/views/applicant/NavigationFragment.html

```diff
@@ -150,8 +150,14 @@
   </svg>
 </span>
 
-<nav th:fragment="guestSessionAlert" th:if="${isGuest and hasProfile}">
-  <section
+<section
+  th:fragment="guestSessionAlert"
+  th:if="${isGuest and hasProfile}"
+  role="alert"
```

Comment:
**issue (blocking):** I think you may be adding this `role="alert"` and `aria-live="polite"` because UX has mentioned it.  `aria-live` only has an effect when there are dynamic changes to that element.  Because these banners exist on page load, it doesn't make sense to give them an `aria-live` attribute or a role of alert.  See this comment, where I explain in more detail:  [comment](https://github.com/civiform/civiform/issues/7686#issuecomment-2477771659)

You can verify this by testing with a screen reader.  If the role of alert actually being applied, the screen reader would go to the banner first, before reading anything else.  For help on screen reader use, see this [wiki page](https://github.com/civiform/civiform/wiki/Testing#screen-reader-testing)